src/Microsoft.AspNetCore.Razor.Language/DefaultRazorIntermediateNodeLoweringPhase.cs

```diff
@@ -96,6 +98,54 @@ protected override void ExecuteCore(RazorCodeDocument codeDocument)
             codeDocument.SetDocumentIntermediateNode(document);
         }
 
+        private void InheritImportedDirectives(DocumentIntermediateNode document, string mainDocumentFilePath)
```

Comment:
I don't like the term *inherit* showing up all of the sudden. We don't use this anywhere else and it's overloaded with other concepts in c#. I'd really prefer this not show up anywhere (comments, code, docs).